beitragsordnung.md

```diff
@@ -2,13 +2,16 @@
 
 Die Mitgliederversammlung des Vereins Welcome Werkstatt e.V. hat am 9.11.2019 folgende Beitragsordnung beschlossen:
 
-1. Alle Vereinsmitglieder – sowohl Fördermitglieder als auch ordentliche Mitglieder mit Stimmrecht – zahlen einen Mitgliedsbeitrag. Der Mitgliedsbeitrag wird monatlich berechnet.
+1. Miedergebühren
+1.1. Alle Vereinsmitglieder – sowohl Fördermitglieder als auch ordentliche Mitglieder mit Stimmrecht – zahlen einen Mitgliedsbeitrag. Der Mitgliedsbeitrag wird monatlich berechnet.
 
-2. Ehrenmitglieder sind von der Beitragszahlung befreit. In Sonderfällen können gemäß Satzung § 6 auch von ordentlichen Mitgliedern mit Stimmrecht begründete Anträge auf Ermäßigung der Beitragshöhe gestellt werden.
+1.2. Ehrenmitglieder sind von der Beitragszahlung befreit. 
```

Comment:
Damit aus diesem Format hier (Markdown) ordentlich eingerückte, nummerierte Listen werden, musst du die Sub-Aufzählungspunkte korrekt einrücken (4 Leerzeichen), und du musst auch die darüberliegende Nummerierung nicht wiederholen, also zum Beispiel:
```suggestion
    2. Ehrenmitglieder sind von der Beitragszahlung befreit. 
```

Reply:
Siehe auch: https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
Ich wollte gerne weg von der fortlaufenden Nummerierung hin zu einer dezimalen Gliederung mit Überschriften. Ich finde es dadurch deutlich Übersichtlicher.

Reply:
> Ich wollte gerne weg von der fortlaufenden Nummerierung hin zu einer dezimalen Gliederung mit Überschriften. Ich finde es dadurch deutlich Übersichtlicher.

Das hab ich schon verstanden. Diese mehrstufige Untergliederung klappt in Markdown aber durch Einrückungen. Jede Einrückung um 4 Leerzeichen fügt eine Gliederungsebene hinzu. Wie das dargestellt wird (also ob dann da 1.1. oder 1. steht oder A. oder 1.A.) das können wir auf der Webseite immer noch modifizieren. Wichtig ist nur, dass es semantisch, also inhaltlich korrekt untergliedert ist.

Reply:
Habe alle Einrückungen korrigiert.